From 5c17a9d3c773347c6edf1ffa3a0c70bc72e157d0 Mon Sep 17 00:00:00 2001
From: Nelson Liu <nelson.liu@mediatek.com>
Date: Thu, 8 Sep 2016 16:22:38 +0800
Subject: [PATCH 06/19] weston: install client-protocol to sysroot for other
 modules access

install *-protocol.c and *-client-protocol.h to sysroot.
Test: test ok

Change-Id: I457140e79c3ef7a4c62f97c955638708129d79bb
Signed-off-by: Nelson Liu <nelson.liu@mediatek.com>
CR-Id: AUTO00000252
---
 Makefile.am | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index f2ae642..98ef915 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -227,7 +227,9 @@ westoninclude_HEADERS =				\
 	shared/matrix.h				\
 	shared/config-parser.h			\
 	shared/zalloc.h				\
-	shared/platform.h
+	shared/platform.h			\
+	protocol/*-protocol.c			\
+	protocol/*-client-protocol.h
 
 if ENABLE_IVI_SHELL
 westoninclude_HEADERS +=				\
-- 
1.9.1

